[2023-05-02 23:08:51,647] INFO _log() L224   * Running on all addresses (0.0.0.0)
   WARNING: This is a development server. Do not use it in a production deployment.
 * Running on http://127.0.0.1:5000
 * Running on http://192.168.15.6:5000 (Press CTRL+C to quit) - call_trace=C:\Users\fadami\AppData\Local\Programs\Python\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-05-02 23:08:51,651] INFO _log() L224   * Restarting with stat - call_trace=C:\Users\fadami\AppData\Local\Programs\Python\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-05-02 23:08:58,329] INFO _log() L224  127.0.0.1 - - [02/May/2023 23:08:58] "[36mGET /openapi/static/js/swagger-ui-standalone-preset.js HTTP/1.1[0m" 304 - - call_trace=C:\Users\fadami\AppData\Local\Programs\Python\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-05-02 23:08:58,330] INFO _log() L224  127.0.0.1 - - [02/May/2023 23:08:58] "[36mGET /openapi/static/css/swagger-ui.css HTTP/1.1[0m" 304 - - call_trace=C:\Users\fadami\AppData\Local\Programs\Python\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-05-02 23:08:58,330] INFO _log() L224  127.0.0.1 - - [02/May/2023 23:08:58] "[36mGET /openapi/static/js/swagger-ui-bundle.js HTTP/1.1[0m" 304 - - call_trace=C:\Users\fadami\AppData\Local\Programs\Python\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-05-02 23:08:58,506] INFO _log() L224  127.0.0.1 - - [02/May/2023 23:08:58] "GET /openapi/openapi.json HTTP/1.1" 200 - - call_trace=C:\Users\fadami\AppData\Local\Programs\Python\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-05-02 23:09:04,901] INFO _log() L224  127.0.0.1 - - [02/May/2023 23:09:04] "DELETE /livro?nome=O%20Alquimista HTTP/1.1" 200 - - call_trace=C:\Users\fadami\AppData\Local\Programs\Python\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-05-02 23:09:06,004] WARNING del_livro() L128  Erro ao deletar livro #'O Alquimista', Produto não encontrado na base :/ - call_trace=C:\Users\fadami\Desktop\projetoApi\app.py L128 
[2023-05-02 23:09:06,005] INFO _log() L224  127.0.0.1 - - [02/May/2023 23:09:06] "[33mDELETE /livro?nome=O%20Alquimista HTTP/1.1[0m" 404 - - call_trace=C:\Users\fadami\AppData\Local\Programs\Python\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-05-02 23:09:33,646] WARNING del_livro() L128  Erro ao deletar livro #'dom casmurro', Produto não encontrado na base :/ - call_trace=C:\Users\fadami\Desktop\projetoApi\app.py L128 
[2023-05-02 23:09:33,646] INFO _log() L224  127.0.0.1 - - [02/May/2023 23:09:33] "[33mDELETE /livro?nome=dom%20casmurro HTTP/1.1[0m" 404 - - call_trace=C:\Users\fadami\AppData\Local\Programs\Python\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-05-02 23:10:03,590] INFO _log() L224  127.0.0.1 - - [02/May/2023 23:10:03] "DELETE /livro?nome=Dom%20Casmurro HTTP/1.1" 200 - - call_trace=C:\Users\fadami\AppData\Local\Programs\Python\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-05-02 23:13:07,544] INFO _log() L224   * Running on all addresses (0.0.0.0)
   WARNING: This is a development server. Do not use it in a production deployment.
 * Running on http://127.0.0.1:5000
 * Running on http://192.168.15.6:5000 (Press CTRL+C to quit) - call_trace=C:\Users\fadami\AppData\Local\Programs\Python\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-05-02 23:13:07,547] INFO _log() L224   * Restarting with stat - call_trace=C:\Users\fadami\AppData\Local\Programs\Python\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-05-02 23:13:16,741] INFO _log() L224  127.0.0.1 - - [02/May/2023 23:13:16] "[32mGET / HTTP/1.1[0m" 302 - - call_trace=C:\Users\fadami\AppData\Local\Programs\Python\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-05-02 23:13:16,787] INFO _log() L224  127.0.0.1 - - [02/May/2023 23:13:16] "GET /openapi/ HTTP/1.1" 200 - - call_trace=C:\Users\fadami\AppData\Local\Programs\Python\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-05-02 23:13:16,998] INFO _log() L224  127.0.0.1 - - [02/May/2023 23:13:16] "[36mGET /openapi/static/images/rapidoc.svg HTTP/1.1[0m" 304 - - call_trace=C:\Users\fadami\AppData\Local\Programs\Python\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-05-02 23:13:17,000] INFO _log() L224  127.0.0.1 - - [02/May/2023 23:13:17] "[36mGET /openapi/static/images/redoc.svg HTTP/1.1[0m" 304 - - call_trace=C:\Users\fadami\AppData\Local\Programs\Python\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-05-02 23:13:17,001] INFO _log() L224  127.0.0.1 - - [02/May/2023 23:13:17] "[36mGET /openapi/static/images/swagger.svg HTTP/1.1[0m" 304 - - call_trace=C:\Users\fadami\AppData\Local\Programs\Python\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-05-02 23:13:17,680] INFO _log() L224  127.0.0.1 - - [02/May/2023 23:13:17] "[32mGET / HTTP/1.1[0m" 302 - - call_trace=C:\Users\fadami\AppData\Local\Programs\Python\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-05-02 23:13:17,687] INFO _log() L224  127.0.0.1 - - [02/May/2023 23:13:17] "GET /openapi/ HTTP/1.1" 200 - - call_trace=C:\Users\fadami\AppData\Local\Programs\Python\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-05-02 23:13:17,710] INFO _log() L224  127.0.0.1 - - [02/May/2023 23:13:17] "[36mGET /openapi/static/images/swagger.svg HTTP/1.1[0m" 304 - - call_trace=C:\Users\fadami\AppData\Local\Programs\Python\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-05-02 23:13:17,727] INFO _log() L224  127.0.0.1 - - [02/May/2023 23:13:17] "[36mGET /openapi/static/images/redoc.svg HTTP/1.1[0m" 304 - - call_trace=C:\Users\fadami\AppData\Local\Programs\Python\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-05-02 23:13:17,739] INFO _log() L224  127.0.0.1 - - [02/May/2023 23:13:17] "[36mGET /openapi/static/images/rapidoc.svg HTTP/1.1[0m" 304 - - call_trace=C:\Users\fadami\AppData\Local\Programs\Python\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-05-02 23:13:19,088] INFO _log() L224  127.0.0.1 - - [02/May/2023 23:13:19] "GET /openapi/swagger HTTP/1.1" 200 - - call_trace=C:\Users\fadami\AppData\Local\Programs\Python\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-05-02 23:13:19,101] INFO _log() L224  127.0.0.1 - - [02/May/2023 23:13:19] "[36mGET /openapi/static/css/swagger-ui.css HTTP/1.1[0m" 304 - - call_trace=C:\Users\fadami\AppData\Local\Programs\Python\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-05-02 23:13:19,104] INFO _log() L224  127.0.0.1 - - [02/May/2023 23:13:19] "[36mGET /openapi/static/js/swagger-ui-bundle.js HTTP/1.1[0m" 304 - - call_trace=C:\Users\fadami\AppData\Local\Programs\Python\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-05-02 23:13:19,106] INFO _log() L224  127.0.0.1 - - [02/May/2023 23:13:19] "[36mGET /openapi/static/js/swagger-ui-standalone-preset.js HTTP/1.1[0m" 304 - - call_trace=C:\Users\fadami\AppData\Local\Programs\Python\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-05-02 23:13:19,285] INFO _log() L224  127.0.0.1 - - [02/May/2023 23:13:19] "GET /openapi/openapi.json HTTP/1.1" 200 - - call_trace=C:\Users\fadami\AppData\Local\Programs\Python\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-05-02 23:13:27,631] INFO _log() L224  127.0.0.1 - - [02/May/2023 23:13:27] "POST /livro HTTP/1.1" 200 - - call_trace=C:\Users\fadami\AppData\Local\Programs\Python\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-05-02 23:13:28,906] WARNING add_livro() L50   Erro ao adicionar livro 'O Alquimista', Produto de mesmo nome já salvo na base :/ - call_trace=C:\Users\fadami\Desktop\projetoApi\app.py L50  
[2023-05-02 23:13:28,907] INFO _log() L224  127.0.0.1 - - [02/May/2023 23:13:28] "[31m[1mPOST /livro HTTP/1.1[0m" 409 - - call_trace=C:\Users\fadami\AppData\Local\Programs\Python\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-05-02 23:13:53,914] INFO _log() L224  127.0.0.1 - - [02/May/2023 23:13:53] "GET /livros HTTP/1.1" 200 - - call_trace=C:\Users\fadami\AppData\Local\Programs\Python\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-05-02 23:14:34,121] INFO _log() L224  127.0.0.1 - - [02/May/2023 23:14:34] "POST /livro HTTP/1.1" 200 - - call_trace=C:\Users\fadami\AppData\Local\Programs\Python\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-05-02 23:15:03,891] INFO _log() L224  127.0.0.1 - - [02/May/2023 23:15:03] "DELETE /livro?nome=Dom%20Casmurro HTTP/1.1" 200 - - call_trace=C:\Users\fadami\AppData\Local\Programs\Python\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-05-02 23:16:15,390] INFO _log() L224  127.0.0.1 - - [02/May/2023 23:16:15] "POST /livro HTTP/1.1" 200 - - call_trace=C:\Users\fadami\AppData\Local\Programs\Python\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-05-02 23:16:15,400] INFO _log() L224  127.0.0.1 - - [02/May/2023 23:16:15] "GET /livros HTTP/1.1" 200 - - call_trace=C:\Users\fadami\AppData\Local\Programs\Python\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-05-02 23:16:40,061] INFO _log() L224  127.0.0.1 - - [02/May/2023 23:16:40] "OPTIONS /livro?nome=Sherlock%20Holmes HTTP/1.1" 200 - - call_trace=C:\Users\fadami\AppData\Local\Programs\Python\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-05-02 23:16:40,072] INFO _log() L224  127.0.0.1 - - [02/May/2023 23:16:40] "DELETE /livro?nome=Sherlock%20Holmes HTTP/1.1" 200 - - call_trace=C:\Users\fadami\AppData\Local\Programs\Python\Python311\Lib\site-packages\werkzeug\_internal.py L224 
[2023-05-02 23:22:18,942] INFO _log() L224  127.0.0.1 - - [02/May/2023 23:22:18] "OPTIONS /livro?nome=O%20Alquimista HTTP/1.1" 200 - - call_trace=C:\Users\fadami\AppData\Local\Programs\Python\Python311\Lib\site-packages\werkzeug\_internal.py L224 
